VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -671,9 +671,11 @@ def do_status_report(self, operation, status, status_code, message, taskId = Non
 
     def write_to_status_file(self, stat_rept_file):
         try:
+            tempStatusFile =  os.path.join(self._context._status_dir, CommonVariables.TempStatusFileName)
             if self._context._status_file:
-                with open(self._context._status_file,'w+') as f:
+                with open(tempStatusFile,'w+') as f:
                     f.write(stat_rept_file)
+                os.rename(tempStatusFile, self._context._status_file)
```

Comment:
make sure this will overwrite when file already exist

Reply:
ya tested that